client/src/components/MainSidebar/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./style.css";
+import { ReactComponent as Logo } from "./../../logo.svg";
+
+export default ({ selected }) => (
+  <aside className="main-side-bar">
+    <a href="/" className="logo-div tab">
+      <Logo />
+    </a>
+
+    <a href="/" className={`tickets tab ${selected === "t" ? "focus" : ""}`}>
```

Comment:
"t" is bad naming.

Reply:
css className should follow the BEM convention.

Reply:
@FarahZaqout 
The letters are the first letters of the words/tabs Tickets, Search and Reports.

And for the classNames these are separate single word class names.